src/parser/mod.rs

```diff
@@ -8144,6 +8144,11 @@ impl<'a> Parser<'a> {
                     Keyword::REPLACE,
                 ])?,
             )))
+        } else if dialect_of!(self is MySqlDialect | GenericDialect)
+            && self.parse_keyword(Keyword::INVISIBLE)
```

Comment:
```suggestion
        } else if self.parse_keyword(Keyword::INVISIBLE)
```
would it make sense to let the parser accept this syntax if it shows up regardless of dialect? we're trying to move away from the `dialect_of` macro and if the parser can be more permissive in this case I think it would be ideal

Reply:
Done.